covid_graphs/covid_graphs/formula.py

```diff
@@ -46,6 +31,41 @@ def get_curve(self, country_report: CountryReport) -> Curve:
         pass
 
 
+@dataclass
+class PolynomialFormula(Formula):
+    a: float
+    exponent: float
+    min_case_count: int
+
+    def get_curve(self, country_report: CountryReport) -> Curve:
+        label = r"$_a \cdot t^{_expon}$"
+        label = label.replace("_a", f"{self.a:.0f}").replace("_expon", f"{self.exponent}")
+
+        start_idx = np.argmax(country_report.cumulative_active >= self.min_case_count)
+        # start_idx marks t=1, so t=0 is one day earlier
+        start_date = country_report.dates[start_idx - 1]
+        display_at_least_until = country_report.dates[-1]
+
+        return Curve(
+            func=lambda x: self.a * (x ** self.exponent),
+            start_date=start_date,
+            display_at_least_until=display_at_least_until,
+            label=label,
+        )
+
+
+def create_label(a: float, tg: float, exponent: float, prefix=""):
```

Comment:
Maybe `_create_atg_label`? (In python, the convention is to prepend `_` for functions that are not supposed to be used outside of the current file.)